db/src/main/resources/db/migration/V1_036__add_county_to_address.sql

```diff
@@ -0,0 +1 @@
+alter table address add column county varchar(500) default 'UK';
```

Comment:
A default won't give existing rows a value. I wonder if it might be better to define a default when we send values to Allpay (and not in the table), but I guess that would make the sql extract quite awkward.

Reply:
Default does give existing rows a value, just tested it locally. 
At the moment will set England as a value for County for AllPay when generating the csv. Surely it would make sense to use the value given if the user provide it.

Reply:
If we're using England at the moment, why are we defaulting to UK here?

Reply:
We decided to use UK as that's a better long term solution. The current allpay instructions are just for private beta, which is only users from England.

Reply:
My thinking was that we'd default to 'UK' if there was no value here, rather than store a default of 'UK' in the table, but I think that would cause problems with our interim sql extract.